multiply_ui/server/context.py

```diff
@@ -2,9 +2,20 @@
 
 from .model import Job
 
+import multiply_data_access.data_access_component
+
 class ServiceContext:
     def __init__(self):
         self._jobs = {}
+        self.data_access_component = multiply_data_access.data_access_component.DataAccessComponent()
+        self._get_mundi_datastore()
+
+    def _get_mundi_datastore(self):
+        for data_store in self.data_access_component._data_stores:
```

Comment:
PEP-8 issue here: `self.data_access_component._data_stores` is a protected member. I'd introduce a read-only property.

Reply:
This is just a workaround anyway; we suggest that the interface allows to select only the datastores needed.

Reply:
Ok, maybe leave comment..